### PR TITLE
Refine package

### PR DESCRIPTION
**Pull request by wep21**
This PR should be merged after https://github.com/tier4/ars408_driver/pull/1.

### CODE REVIEW

File: include/ars408_ros/ars408_commands.hpp [hunk 1]
```diff
@@ -0,0 +1,673 @@
+// Copyright 2021 Perception Engine, Inc. All rights reserved.
```
Comment (kenji-miyake) [inline include/ars408_ros/ars408_commands.hpp:1]: Did you use `git mv`? :thinking: 
It seems not to be recognized as the same file.
image

Comment (kenji-miyake) [reply include/ars408_ros/ars408_commands.hpp:1]: Hmm, perhaps we should split PRs to show the diff correctly... :pleading_face:

Comment (kenji-miyake) [reply include/ars408_ros/ars408_commands.hpp:1]: I'm okay if you want to merge as it is.

Comment (wep21) [reply include/ars408_ros/ars408_commands.hpp]: I split PR. https://github.com/tier4/ars408_driver/pull/3

Comment (kenji-miyake) [reply include/ars408_ros/ars408_commands.hpp]: God.

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**